.eslintignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
-test-reports
\ No newline at end of file
+test-reports
+test/apps/angular-v*
```

Comment:
Why ignore test apps from eslint?

Reply:
The linter was throwing errors which were being ignored. The test apps do not contain a copy of `@okta/angular` during build time, so the linter flags it

```bash
Start lint
$ eslint --ext .js,.ts  .

/root/okta/okta-angular/test/apps/angular-v12/src/app/app.component.spec.ts
  7:8  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v12/src/app/app.component.ts
  15:49  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v12/src/app/app.module.ts
  26:8  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v12/src/app/protected.component.ts
  14:27  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v12/src/app/public.component.ts
  14:27  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v12/src/app/sessionToken-login.component.ts
  14:27  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v13/src/app/app.component.spec.ts
  7:8  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v13/src/app/app.component.ts
  15:49  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved
```

Reply:
Possible to ignore this specific rule in test apps eslintrc other than ignore all lint checks?

Reply:
looks like there is a setting to ignore specific packages
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unresolved.md#ignore